chain-core/src/init/address.rs

```diff
@@ -174,25 +177,22 @@ impl RedeemAddress {
 impl CroAddress<RedeemAddress> for RedeemAddress {
     fn to_cro(&self) -> Result<String, CroAddressError> {
         let checked_data: Vec<u5> = self.0.to_vec().to_base32();
-        match super::network::get_network() {
-            super::network::Network::Testnet => {
-                let encoded = bech32::encode("crts", checked_data).expect("bech32 crms encoding");
-                Ok(encoded.to_string())
-            }
-            super::network::Network::Mainnet => {
-                let encoded = bech32::encode("crms", checked_data).expect("bech32 crms encoding");
-                Ok(encoded.to_string())
-            }
-        }
+        let encoded = bech32::encode(super::network::get_bech32_human_part(), checked_data)
+            .expect("bech32 crms encoding");
```

Comment:
Could this encoding return an error instead fo panicking (or would it ever panic / return an error?)?

Reply:
This will only panic when human readable part is invalid and I assume we already have a static assertion for that (@leejw51crypto)?

[Documentation](https://docs.rs/bech32/0.7.1/bech32/fn.encode.html)